src/planscape/conditions/registry.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+import subprocess
+from django.conf import settings
+from conditions.models import Condition, ConditionRaster
+from base.condition_types import ConditionScoreType
+from utils.cli import psql, psql_pipe, raster2pgpsql
+
+def get_raster_path(condition):
```

Comment:
I waffled on whether or not this belongs in raster_utils or not.

nit of a nit: maybe call this generate_raster_path_from_condition or something like that - so that it's not implying it's simply fetching a field from an opaque struct.  Same goes for get_tile_name.

TODO: add tests for these; they're simple enough but are setting a pretty critical naming convention that can't be changed by the casual coder later.